internal/pkg/deploy/cloudformation/project.go

```diff
@@ -439,3 +439,45 @@ func (cf CloudFormation) waitForStackSetOperation(stackSetName, operationID stri
 		time.Sleep(3 * time.Second)
 	}
 }
+
+// DeleteProject deletes all project specific StackSet and Stack resources.
+func (cf CloudFormation) DeleteProject(name string, accounts, regions []string) error {
+	if _, err := cf.client.DeleteStackInstances(&cloudformation.DeleteStackInstancesInput{
+		Accounts:     aws.StringSlice(accounts),
+		RetainStacks: aws.Bool(false),
+		Regions:      aws.StringSlice(regions),
+		StackSetName: aws.String(fmt.Sprintf("%s-infrastructure", name)),
+	}); err != nil {
+		return err
+	}
+
+	maxAttempts := 10
```

Comment:
Shall we make this a constant and the sleep time as well?  This will make it easier when we want to adjust them in the future. (10 times is not enough or 30 seconds is too short)